packages/usa-button/src/styles/_usa-button.scss

```diff
@@ -51,6 +51,12 @@ $button-stroke: inset 0 0 0 units($theme-button-stroke-width);
   &[aria-disabled="true"] {
     @include button-disabled;
   }
+
+  @media (forced-colors: active) {
+    &:not(.usa-button--unstyled, .usa-focus) {
+      @include u-forced-color-outline;
```

Comment:
Setting the outline property here is risky because it interferes with the `:focus` state. What do you think about using border here instead so that it has both the border and outline on focus?

Reply:
I tried swapping the mixin's outline for border and it looks like the desired affect is consistent across all the components that use it and it doesn't interfere with `:focus` state. I believe the only reason I had them originally set to outline was that I could declare `outline: 2px solid transparent` without a media query and it still wouldn't affect the component's normal border. Since we're using the forced-colors media query, I think this solution should do the trick!

<img width="506" alt="image" src="https://user-images.githubusercontent.com/25211650/221969531-0bbd4f5c-5e16-4715-8236-7e3a01bb3087.png">

<img width="506" alt="image" src="https://user-images.githubusercontent.com/25211650/221969839-13d11b47-c874-486d-969c-2a93f4532551.png">

<img width="506" alt="image" src="https://user-images.githubusercontent.com/25211650/221969885-02d1505d-3a98-4014-b920-bdf493516a64.png">

I did notice that other focus and hover outlines seem to be not working on Date Picker on our develop branch. Did we change the background svg's since the high contrast work?

Reply:
Resolved in [60537b8](https://github.com/uswds/uswds/pull/5147/commits/60537b8d81d5a7c043c1291c74526722a35d8ffa)